design-documents/graph-ql/coverage/add-items-to-cart.md

```diff
@@ -0,0 +1,49 @@
+**Overview**
+
+As a Magento developer, I need to manipulate the shopping cart via GraphQL so that I can build basic ecommerce experiences for shoppers on the front-end using only GraphQL.
```

Comment:
> for shoppers on the front-end using only GraphQL.

This is a big nitpick, so...sorry!

I think it's super important that we avoid language that couples the concept of GraphQL in Magento to the front-end specifically.

I know it's a small thing, but we want to make sure everyone has the mindset that this is an API for _any_ external consumer.

Reply:
Please suggest how to stress, that GraphQL in Magento is intended for store front scenarios only, not for admin ones.

Reply:
How about:

```
As a Magento developer, I need to manipulate the shopping cart via GraphQL so that I can programmatically create orders on behalf of a shopper.
```

Thoughts?